ui/DEBIAN/postinst

```diff
@@ -31,13 +31,13 @@ if [ "$1" == "configure" ] ; then
       [ -d "$LOG_DIR" ] && chown -R $USER:$GROUP $LOG_DIR && chmod 777 $LOG_DIR
     fi
 
-	cd /usr/share
-	mkdir -p $PAC
-	tar xzf $PAC.tgz -C $PAC
```

Comment:
Was package archive created earlier in make deb script? Wanted to understand the purpose of this line if it had any